Ghidra/Features/Decompiler/src/decompile/cpp/printc.cc

```diff
@@ -2867,6 +2879,33 @@ void PrintC::emitBlockSwitch(const BlockSwitch *bl)
   popMod();
 }
 
+/// \brief Emits an opening brace according to the indentation style selected in options
+int4 PrintC::emitFormatedStartBrace(int4 indent)
+
+{
+  if (option_indentationStyle == indentation_style_allman) {
+  	if (NULL != indent) {
```

Comment:
indent is an int4, not pointer, would be better as `if (0 != indent)` or simply `if (indent)`